src/convertToObject.js

```diff
@@ -12,7 +13,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .replace(/\s+/g, ' ')
+    .split(';')
+    .map(propertie => propertie.trim())
+    .filter(propertie => propertie)
+    .map(propertie => {
```

Comment:
why `map`? you don't save result array. to just iterate an array there is another method